cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
Hi @dmitrypekar, you should 2) integrate new Truncate job in a new RestoreClusterJobHandler adding all those complex coordination logic to invoke Truncate first and then Restore for each node